packages/terra-notification-dialog/src/terra-dev-site/doc/example/ActionOrderExample.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import Button from 'terra-button';
+import NotificationDialog from 'terra-notification-dialog';
+
+const NoVariantNotificationDialog = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [buttonOrder, setButtonOrder] = useState();
+
+  const handleCloseModal = (actionClicked) => {
+    alert(`You clicked the ${actionClicked} action.`); // eslint-disable-line no-alert
```

Comment:
I don't have a huge opinion on this, but instead of using an alert we could update a div or p on the dom to indicate the action was taken.

Reply:
I just removed it; It didn't add much value IMO: https://github.com/cerner/terra-framework/pull/1105/commits/440c28411da12f4c7b6d14c932c0437e8a289e63#diff-cf4f3b43a81f8e1ceb70734455bd7672L10